### DISCUSSION TITLE
having troubles with the setup manager

### DISCUSSION THREAD

**Discussion by blackpandan** [hrfee/jfa-go#139] [Q&A]
hello I am having trouble setting this up on the web UI with jellyfin

**Comment by hrfee** [comment]
What issues are you having specifically?

**Reply by blackpandan** [reply]
```jfa-go systemd```
does not generate service file
is there a way I can create this manually?

**Reply by blackpandan** [reply]
This is a sample service file i created

```
[Unit]
Description=Jfa-go service 
Requires=jellyfin.service
After=jellyfin.service
StartLimitIntervalSec=0


[Service]
Type=simple
Restart=always
RestartSec=1
User=root
ExecStart=/usr/local/bin/jfa-go

[Install]
WantedBy=multi-user.target
```

**Reply by hrfee** [reply]
That command is broken on 0.3.7 but should work on newer unstable releases. Your own service file looks fine though.

**Comment by blackpandan** [comment]
and please how can i configure the address jfa-go redirects to after account creation

**Reply by hrfee** [reply]
Have you been through the setup process yet?

**Comment by blackpandan** [comment]
yes i finished setting it up

**Reply by hrfee** [reply]
You can set the public jellyfin address in Settings > Jellyfin > Public Address

**Reply by Naomarn** [reply]
Is this where users will be redirected to after clicking Create Account? Mine turns red and does go anywhere or do anything
